yb-voyager/src/srcdb/source.go

```diff
@@ -315,6 +336,93 @@ func (source *Source) extrapolateDSNfromSSLParams(DSN string) string {
 	return DSN
 }
 
+func (source *Source) getPgDumpSchemaArgsFromFile() string {
+	pgDumpArgsFilePath := filepath.Join("/", "etc", "yb-voyager", "pg_dump-args.ini")
+	if !utils.FileOrFolderExists(pgDumpArgsFilePath) {
+		return ""
+	}
+	log.Infof("Using pg_dump arguments file: %s", pgDumpArgsFilePath)
+	pgDumpArgsFile, err := os.ReadFile(pgDumpArgsFilePath)
+	if err != nil {
+		utils.ErrExit("Error while reading pg_dump arguments file: %v", err)
+	}
+
+	tmpl, err := template.New("pg_dump_args").Parse(string(pgDumpArgsFile))
+	if err != nil {
+		utils.ErrExit("Error while parsing pg_dump data arguments: %v", err)
+	}
+
+	var output bytes.Buffer
+	err = tmpl.Execute(&output, pgDumpArgs)
+	if err != nil {
+		utils.ErrExit("Error while preparing pg_dump data arguments: %v", err)
+	}
+
+	iniData, err := ini.Load(output.Bytes())
+	if err != nil {
+		utils.ErrExit("Error while ini loading pg_dump arguments file: %v", err)
+	}
+	section := iniData.Section("schema")
+	args := ""
+	for _, key := range section.Keys() {
+		if key.Value() == "false" {
+			continue
+		}
+		if key.Value() == "true" {
+			args += fmt.Sprintf(" %s", key.Name())
+		} else {
+			if key.Name() == "--schema" {
```

Comment:
Don't see a difference in the if-else blocks 🤔 What is the intention here?

Reply:
the schema names are comma separate multiple names. Need to provided within quotes. 
I tried other ways like putting the quotes inside `.ini` file or passing it as a value but they got considered as a part of string.